dance/transforms/normalize.py

```diff
@@ -250,15 +250,14 @@ def _get_idx_dict(self, data) -> List[Dict[str, List[int]]]:
 
         return idx_dict
 
-    def __call__(self, data):
+    def __call__(self, data: Data):
         if isinstance(data.data.X, sp.spmatrix):
             self.logger.warning("Native support for sparse matrix is not implemented yet, "
                                 "converting to dense array explicitly.")
             data.data.X = data.data.X.A
-
         # idx_dict = self._get_idx_dict(data)
         # for name, idx in idx_dict.items():
-        selected_data = data.data.copy()
+        selected_data = data.data
```

Comment:
Can we add an option for deciding whether to overwrite the original `X` matrix or saving it as a new layer? It might be good to have the option to keep the original `X` and save the normalized matrix to `adata.layers[self.out]`.

Reply:
I'm not sure because there is saveRaw function that helps save raw data

Reply:
Ok fair enough, I'll also keep thinking about it